### ISSUE TITLE
 Quote icon [Icon Tool]

### ISSUE THREAD

**Issue by Juliamirjami** [OfficeDev/office-ui-fabric-react#10405]
Hello!
We are making a rich text editor tool. The quote icon is tiny compared to the other editor icons side by side. Can it be amended to fit the others? I realised in Teams app editor the quote marks had been scaled up. Looks more visible/accessible.

Best, Julia

**Comment by natalieethell** [comment]
@Jahnp Could you take a look at this one when you have a moment?

**Comment by Jahnp** [comment]
Thanks for tagging me @natalieethell !

Thanks for bringing this to our attention, @Juliamirjami ! To help understand which quote icon you're referring to, would you mind sharing an image, the friendly name, and/or Unicode value you're using?

I assume you're referring to this icon, `E9B1`/`RightDoubleQuote`:
![image](https://user-images.githubusercontent.com/307118/64734585-c1536c00-d49b-11e9-9c13-16348a968f67.png)

Before requesting any new glyphs to the icons team, do any of these icons look like better substitutes for your scenario? I'm guessing 1, "Quote" is probably the closest, since the others are essentially identical to the `RightDoubleQuote` we have today.
![image](https://user-images.githubusercontent.com/307118/64734725-0aa3bb80-d49c-11e9-8cc3-7be8241d4dc0.png)

**Comment by Juliamirjami** [comment]
Hi Peter

We are using the icons tool (link below) and E9B1 is the ‘quote’ icon we found. Find attached comparison of how tiny the quote looks compared to other icons. I realised that in Teams text editor’s quote icon has been made similar size to others which feels more natural.

Thanks for your help!
Julia

https://uifabricicons.azurewebsites.net


[A picture containing electronics, outdoor, sitting  Description automatically generated]
